platform/resources/runJobOnWorker.groovy

```diff
@@ -0,0 +1,7 @@
+import hudson.model.*
+
+JOB_TO_RUN = build.environment.get('JOB_TO_RUN')
+ghprbActualCommit = build.environment.get('ghprbActualCommit')
+WORKER_LABEL = build.environment.get('WORKER_LABEL')
+
+build(JOB_TO_RUN, ghprbActualCommit: ghprbActualCommit, WORKER_LABEL: WORKER_LABEL)
```

Comment:
@estute - I'm a little confused here. Is this a hudson build() method call? Does this line work?
I don't see this pattern in other scripts, tho I could be missing it. Could you pls include a link to the doc or another example?